doc/source/changes/version_0_21.rst.inc

```diff
@@ -0,0 +1,110 @@
+  (closes :issue:`67`).
+
+    >>> arr = ndtest((2, 3))
+    >>> arr
+    a\\b | b0 | b1 | b2
+     a0 |  0 |  1 |  2
+     a1 |  3 |  4 |  5
+    >>> row = Axis('row', ['r0', 'r1'])
+    >>> column = Axis('column', ['c0', 'c1', 'c2'])
+
+    Replace one axis (expect 2 arguments)
+
+    >>> arr.replace_axes(x.a, row)
+    row\\b | b0 | b1 | b2
+       r0 |  0 |  1 |  2
+       r1 |  3 |  4 |  5
+
+    Replace several axes (keywords, list of tuple or dictionary)
+
+    >>> arr.replace_axes(a=row, b=column)
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+    >>> arr.replace_axes([(x.a, row), (x.b, column)])
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+    >>> arr.replace_axes({x.a: row, x.b: column})
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+
+    Replace all axes (list of axes or AxisCollection)
+
+    >>> arr.replace_axes([row, column])
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+    >>> arr2 = ndrange([row, column])
+    >>> arr.replace_axes(arr2.axes)
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+
+* implemented from_string() method to create an array from a string:
+
+    >>> from_string('''nat\\sex, M, F
+    ...                BE,       0, 1
+    ...                FO,       2, 3''')
+    nat\sex | M | F
+         BE | 0 | 1
+         FO | 2 | 3
+    >>> from_string('''age,nat\\sex, M, F
+    ...                0,  BE,       0, 1
+    ...                0,  FO,       2, 3
+    ...                1,  BE,       4, 5
+    ...                1,  FO,       6, 7''')
+    age | nat\sex | M | F
+      0 |      BE | 0 | 1
+      0 |      FO | 2 | 3
+      1 |      BE | 4 | 5
+      1 |      FO | 6 | 7
+
+Miscellaneous improvements
+--------------------------
+
+* readme.rst + doc : add installation instructions (closes :issue:`101`).
+
+* allowed to use regex in split_axis method (closes :issue:`106`):
+
+    >>> combined = ndrange('a_b = a0b0..a1b2')
+    >>> combined
+        a_b | a0b0 | a0b1 | a0b2 | a1b0 | a1b1 | a1b2
+            |    0 |    1 |    2 |    3 |    4 |    5
+    >>> combined.split_axis(x.a_b, regex='(\w{2})(\w{2})')
+        a\\b | b0 | b1 | b2
+         a0 |  0 |  1 |  2
+         a1 |  3 |  4 |  5
+
+* allowed multiplication (__matmul__ or @ operator) between matrices with dimension != 2 (closes :issue:`122`).
+
+* implemented Axis.by() which is equivalent to axis[:].by()
+
+* viewer : automatically display plots done in the viewer console in a separate window
+  (unless "%matplotlib inline" is used)
+
+Fixes
+-----
+
+* viewer: allow changing the number of displayed digits even for integer arrays as that makes sense when using
+  scientific notation (closes :issue:`100`).
+
+* viewer : make shortcuts work even when the focus is not on the array editor widget
```

Comment:
I don't consider this a bug fixed, but an improvement, but well this is subjective :)